textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -42,12 +47,12 @@ public static IndexableField getLuceneField(FieldType fieldType,
             String fieldName, Object fieldValue) {
         IndexableField luceneField = null;
         switch(fieldType){
-            case STRING:
+	        case STRING:
                 luceneField = new org.apache.lucene.document.StringField(
                         fieldName, (String) fieldValue, Store.YES);
                 break;
             case INTEGER:
-                luceneField = new IntField(
+                luceneField = new org.apache.lucene.document.IntField(

```

Comment:
Shouldn't `org.apache.lucene.document.IntField` be imported and then used instead of using the entire package name? @JavierJia @chenlica